src/tdastro/sources/periodic_variable_star.py

```diff
@@ -0,0 +1,243 @@
+from abc import ABC, abstractmethod
+
+import astropy.units as u
+import numpy as np
+from astropy.modeling.physical_models import BlackBody
+
+from tdastro.sources.periodic_source import PeriodicSource
+
+PARSEC_TO_CM = (1 * u.pc).to_value(u.cm)
```

Comment:
Let's create a `src/tdastro/constant.py` (or some other name) for this.

Reply:
Thank you, I'll do it. Also, I'd like us to have some decision about units we use across the code.